web/components/input/text/TextInput/TextInput.sass

```diff
@@ -52,5 +50,6 @@ div.text-input
 
   p
     margin-top: 4px
+    margin-bottom: -18px
```

Comment:
this seems unfortunately hacky 🤔 what caused this to be needed?

Reply:
i forgot lol